model.py

```diff
@@ -128,7 +146,7 @@ def __init__(self, user_id, task_name, duration_estimate):
     def __repr__(self):
         """Provide helpful representation when printed."""
 
-        return f"<Task task_id={self.task_id} task_name={self.task_name} user_id={self.user_id}>"
+        return f"<Task {self.id} | {self.name} for user {self.user_id}>"
```

Comment:
This is a good example of making your `repr` much more specialized and useful. We want to know task ids and whos task it is at a glance. So we use plain english and keep it brief.